src/schnet/CudaCFConv.cu

```diff
@@ -260,11 +260,11 @@ const float* CudaCFConv::ensureOnDevice(const float* arg, float*& deviceMemory,
     return (const float*) attrib.devicePointer;
 }
 
-__device__ float cutoffFunction(float r, float rc) {
+__device__ float cutoffFunction2(float r, float rc) {
```

Comment:
Why renaming these functions?

Reply:
Name collision when linking due to functions with the same names in `CudaANISymmetryFunctions.cu`

Reply:
Open to suggestions for a better naming scheme.

Reply:
How about adding the `static` keyword so it knows those functions are local to this file and shouldn't be exported?

Reply:
Fixed!